syntax = "proto3";

package jio.brain.proto.knowledge.api.data;

option java_package = "com.jio.brain.proto.knowledge.api.data";
option java_multiple_files = true;

import "jio/brain/proto/base/status.proto";

service GetEntityContextService {
    rpc serve(GetEntityContextRequest) returns (GetEntityContextResponse) {}
}

message GetEntityContextRequest {
    uint32 entity_id = 1; 
}

message GetEntityContextResponse {
    repeated EntityContext entity_context = 1;
    jio.brain.proto.base.BrainStatus status = 2;
}

message EntityContext {
    string context_type = 1;
    uint32 id = 2;
    uint32 maximum = 3;
    uint32 minimum = 4;
}